ppr-ui/src/composables/mhrRegistration/useNewMhrRegistration.ts

```diff
@@ -127,6 +132,13 @@ export const useNewMhrRegistration = () => {
     }
   }
 
+  const resetLocationInfoFields = (location: MhrLocationInfoIF): MhrLocationInfoIF => {
+    Object.entries(location).forEach(([key]) => {
+      location[key] = ''
+    })
+    return location
+  }
```

Comment:
Just keep in mind this only works because all the properties are of String Type.
If we were to add types outside of this, would require a bit more finesse 👍